rules/61-autosuspend-manual.rules

```diff
@@ -0,0 +1,20 @@
+# This udev rule is for any devices that should enter automatic suspend
+# but are not already included in generated rules from ChromeOS via
+# tools/make-autosuspend-rules.py
+#
+
+ACTION!="add", GOTO="autosuspend_manual_end"
+SUBSYSTEM!="usb", GOTO="autosuspend_manual_end"
+
+SUBSYSTEM=="usb", GOTO="autosuspend_manual_usb"
+
+# USB rules
+LABEL="autosuspend_manual_usb"
+GOTO="autosuspend_manual_end"
+
+# Enable autosuspend
+LABEL="autosuspend_manual_enable"
+TEST=="power/control", ATTR{power/control}="auto", GOTO="autosuspend_manual_end"
```

Comment:
This seems a bit awkward without any rules like this, but if the plan is to add some more device here soon, than it should be OK.

Reply:
Hm, I don't really see the need for a separate udev rule.
Could we update `gen_autosuspend_rules.py` to simply dump a static list of such devices and attach that to the autogenerated list?

Reply:
I guess we could. But this approach is very simple, and it's always easier for people to submit patches against a static file, instead of a generated one, where you need to modify something else to get the desired result. If it turns out that the list is really short, we could merge it as you suggest.

Reply:
gen_autosuspend_rules.py is supposed to stay exactly in sync with chromeos to make it easier to maintain. If we start adding manual stuff it will need to be merged whenever new chromeos changes need to come in.

Reply:
And regarding the first comment; yes the plan is once @jwrdegoede gets an opportunity to test some of the WWAN modems he has on hand he'll grow the list of manual stuff some.